crest/Assets/Crest/Crest/Shaders/OceanGlobals.hlsl

```diff
@@ -17,6 +17,7 @@ float _TexelsPerWave;
 float3 _OceanCenterPosWorld;
 float _SliceCount;
 float _MeshScaleLerp;
+float _ClipByDefault;
```

Comment:
The TODO at the top of this file says we should prefix these with Crest.

Reply:
Good point, fixed